cmd/lakefs/cmd/migrate.go

```diff
@@ -15,23 +18,41 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print current migration version and available version",
 	Run: func(cmd *cobra.Command, args []string) {
-		logging.Default().Info("Nothing to do, DB migrate is deprecated")
+		logger := logging.Default()
+		cfg := loadConfig()
+
+		ctx := cmd.Context()
+		kvParams := cfg.GetKVParams()
+		kvStore, err := kv.Open(ctx, kvParams)
+		if err != nil {
+			logger.WithError(err).Fatal("Failed to open KV store")
+		}
+		defer kvStore.Close()
+
+		version, err := kv.GetDBSchemaVersion(ctx, kvStore)
```

Comment:
If it's a new installation before setup - maybe we should return a different error on 'not found', such as 'KV not initialized' or something similar?

Reply:
Perhaps use ValidateSchemaVersion

Reply:
Updated base don modified ValidateSchema.
Will do followup PR to remove additional DB which we no longer use.
And enable retry pg connect, found it missing in the new kv implementation.